content/en/continuous_integration/quality_gates/_index.md

```diff
@@ -0,0 +1,170 @@
+they do not block the pipeline when they fail.
+6. Select the time window over which the query will be evaluated.
+7. Select a rule name that describes the rule that you are creating.
+8. Click **Save Rule**.
+
+
+## Datadog-ci gate evaluate
+
+To use quality gates, [`datadog-ci`][7] version should be higher or equal than `2.16.0`.
+
+You can invoke the Quality Gates evaluation by calling the [`datadog-ci gate evaluate`][4] command.
+
+This command:
+
+1. Retrieves all the rules that have [rules scopes](#rule-scope) and [custom scopes](#custom-scope) matching the current pipeline context (repository, branch, or custom scope passed in the command).
+2. Evaluates all the matching rules.
+3. If one or more blocking rules fail, the command fails as well, blocking the pipeline.
+
+<div class="alert alert-danger"><strong>Note:</strong> for the command to work properly, it's important that the events (tests, static analysis violations)
+are sent to Datadog <strong>before</strong> the <code>datadog-ci gate evaluate</code> command is executed.
+Otherwise, the rules might have an incorrect behavior due to the absence of the events.
+</div>
+
+The command requires the `DATADOG_API_KEY` and `DATADOG_APP_KEY` environment variables to point to your [Datadog API Key][5]
+and [Datadog Application Key][6]. Also, you need to set the `DD_BETA_COMMANDS_ENABLED` environment
+variable as `true`. Optionally, you can specify the `DATADOG_SITE` environment variable to point to a specific datadog site (default value is `datadoghq.com`).
+
+{{< code-block lang="shell" >}}
+DD_BETA_COMMANDS_ENABLED=true DATADOG_SITE={{< region-param key="dd_site" >}} DATADOG_API_KEY=<API_KEY> DATADOG_APP_KEY=<APP_KEY> datadog-ci gate evaluate
+{{< /code-block >}}
+
+The behavior of the command can be modified using the following flags:
+- --fail-on-empty: when this flag is specified, the command fails if no matching rules were found in Datadog
+based on the current command scope. By default, the command succeeds.
+- --fail-if-unavailable: when this flag is specified, the command fails if one or more rules could not be evaluated because of an internal issue.
+By default, the command succeeds.
+- --no-wait: by default, the command waits a certain amount of time for the events (tests, static analysis violations) to arrive to Datadog.
+This step is important as it makes sure that the events are queryable in Datadog before the rules are executed,
+avoiding incorrect evaluation. If, in your pipeline, the job containing the `datadog-ci gate evaluate` command is
+called several minutes after the related events are sent to Datadog, you could skip this waiting time by specifying the `--no-wait` flag.
+Note that, if used incorrectly, this flag might result in inaccurate rule evaluations.
+
+[Custom scopes](#custom-scope) can be added by using the following option one or more times: **--scope**:
```

Comment:
Here the double hyphen on --scope is also being replaced by an 'en dash': –scope

Reply:
Yes, we have a [thread](https://github.com/DataDog/documentation/pull/18954#discussion_r1266978293) open. I will resolve this one, please comment there in case 🙏